tests/functional/adapter/constraints/test_bigquery_constraints.py

```diff
@@ -0,0 +1,199 @@
+import pytest
+import re
+import json
+from dbt.tests.util import (
+    run_dbt,
+    get_manifest,
+    run_dbt_and_capture
+)
+from dbt.tests.adapter.constraints.test_constraints import (
+    TestModelLevelConstraintsEnabledConfigs,
+    TestModelLevelConstraintsDisabledConfigs,
+    TestSchemaConstraintsEnabledConfigs,
+    TestModelLevelConstraintsErrorMessages
+)
+
+
+my_model_sql = """
+{{
+  config(
+    materialized = "table"
+  )
+}}
+select
+  1 as id,
+  'blue' as color,
+  cast('2019-01-01' as date) as date_day
+"""
+
+my_model_error_sql = """
+{{
+  config(
+    materialized = "table"
+  )
+}}
+select
+  null as id,
+  'blue' as color,
+  cast('2019-01-01' as date) as date_day
+"""
+
+model_schema_yml = """
+version: 2
+models:
+  - name: my_model
+    config:
+      constraints_enabled: true
+    columns:
+      - name: id
+        data_type: integer
+        description: hello
+        constraints: 
```

Comment:
```suggestion
        constraints: ["not null"]
```

This is the official way to define constraints